### ISSUE TITLE
Causal model

### ISSUE THREAD

**Issue by sunmengnan** [open-mmlab/mmpose#1052]
could you provide model in Causal mode in Videopose3d ?

**Comment by ly015** [comment]
We have this [page](https://github.com/open-mmlab/mmpose/issues/383) to collect model zoo requests. Could you please add your request there following the template? Thank you.

**Comment by sunmengnan** [comment]
Are these 2d-3d Videopose3d models trained by yourself? Or downloaded

**Comment by jin-s13** [comment]
All models in the model zoo are trained using MMPose.

**Comment by sunmengnan** [comment]
Thx, are these video3dpose models trained by supervised setting, or semi-supervised setting？

**Comment by ly015** [comment]
We provided both. Please refer to [videopose3d_h36m.md](https://github.com/open-mmlab/mmpose/blob/master/configs/body/3d_kpt_sview_rgb_vid/video_pose_lift/h36m/videopose3d_h36m.md) for details.

**Comment by sunmengnan** [comment]
One thing makes me confused is that why only use 10% of S1 for supervised traning in semi-supervised setting? As semi-supervised training use S5,6,7,8., can  we use like 90% of S1?

**Comment by ly015** [comment]
We follow the experiment settings in the paper. You could conduct experiments with your own settings by modifying the config file.

**Comment by sunmengnan** [comment]
Have you tested these models on some other indoor videos? when I used these models tested on some videos, when performing the same action(such as walking), the depth joints are not stable, sometimes people stand nearer the camera, the more deeper.

**Comment by ly015** [comment]
This could be caused by the gap between training data and the testing scenario. The training data (e.g. human3.6m) is usually collected in constraint settings (environment, human identities, action, camera parameters ...). So the model may not be able to perform well under user settings.

It helps to improve the accuracy and stability if you could normalize your data (e.g. bbox scale) to a similar distribution of the training data.
